csharp/test/Apache.Arrow.Tests/Decimal128ArrayTests.cs

```diff
@@ -41,7 +40,7 @@ public void AppendThenGetGivesNull()
                     var array = builder.Build();
 
                     Assert.Equal(3, array.Length);
-                    Assert.Equal(array.Data.Buffers[1].Length, array.ByteWidth * 3);
+                    Assert.Equal(array.Data.Buffers[1].Length, array.ByteWidth * 3 + array.ByteWidth - 1);
```

Comment:
What is the purpose of this PR? It seems like it makes this computation unnecessarily complex. Decimals are either 16 or 32 bytes per the Arrow specification.

Reply:
yeah indeed seeing cpp implementation its fixed

I saw in ParquetSharp implementation returning a Decimal128Type with 9 byteWidth for precision = 15 and scale = 5 which was different from 16